### PR TITLE
add margin

### PR DESCRIPTION
**Pull request by Made-Code**
add margin 2px

### CODE REVIEW

**Comment by Made-Code** [comment]
thank you

File: style.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
 body {
   font-family: 'Courier New', Courier, monospace;
   font-size: medium;
-}
\ No newline at end of file
+  margin: 2px;
```
Comment (Made-Code) [inline style.css:4]: good change

**Comment by MadeWelly** [comment]
thank you

File: style.css [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
+  paddign:2px;
```
Comment (MadeWelly) [inline style.css:5]: good

**Pull request merged**